malwarefront/src/components/AttributeUpdate.js

```diff
@@ -4,7 +4,7 @@ import Autocomplete from 'react-autocomplete';
 import _ from "lodash";
 
 import api from "@malwarefront/api";
-import { Alert, ConfirmationModal, ObjectLink } from "@malwarefront/ui";
+import {  ConfirmationModal, ObjectLink, View } from "@malwarefront/ui";
```

Comment:
Too many spaces in `import`
```suggestion
import { ConfirmationModal, ObjectLink, View } from "@malwarefront/ui";
```